theano/gof/opt.py

```diff
@@ -1770,6 +1771,8 @@ def process_node(self, fgraph, node, lopt=None):
                 raise
         if replacements is False or replacements is None:
             return False
+        if replacements is True:
+            return True
```

Comment:
It mostly mean, I did some change directly on the fgraph. I agree, this should be documented. But before that, we should agree that this is acceptable.